pg_backup_api/setup.py

```diff
@@ -46,8 +40,8 @@
     author="EnterpriseDB",
     author_email="barman@enterprisedb.com",
     url="http://www.pgbarman.org/",
-    keywords=["OpenAPI", "Postgres Backup REST API"],
-    python_requires=">=3.6",
+    keywords=["Postgres Backup REST API"],
+    python_requires=">2.7",
```

Comment:
Not sure supporting 2.7 for pg-backup-api this is the best direction. Unless we have a real need for it we should keep to 3.6 and more.

Reply:
We need to support 2.7 for CentOS7 and SLES12 since Barman is packaged for python 2.7 on those platforms. Ideally we'd move those to 3.7 (since 3.6 is now EOL) and then pg-backup-api would just require 3.7 or greater, however this feels like a bigger problem so I'd rather tackle it after addressing pg-backup-api and its packaging/dependency issues.